.travis.yml

```diff
@@ -1,53 +1,19 @@
 ---
-sudo: false
-language: ruby
-cache: bundler
+sudo: required
+services: docker
+addons:
+  apt:
+    packages:
+    - docker-ce
 env:
-- INTEGRATION=false
-- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=1.7.6 TEST_DEBUG=true
-- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
-- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=5.6.13 TEST_DEBUG=true
-- DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=6.0.0 TEST_DEBUG=true
-- DISTRIBUTION=oss INTEGRATION=true ES_VERSION=6.6.0 TEST_DEBUG=true
-- DISTRIBUTION=default INTEGRATION=true ES_VERSION=6.6.0 TEST_DEBUG=true
-- DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-6 TEST_DEBUG=true
-- DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-7 ARCH=LINUX TEST_DEBUG=true
-- DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-8 ARCH=LINUX TEST_DEBUG=true
-- DISTRIBUTION=legacyxpack SECURE_INTEGRATION=true INTEGRATION=true ES_VERSION=5.6.13
-  TEST_DEBUG=true
-- DISTRIBUTION=default SECURE_INTEGRATION=true INTEGRATION=true ES_VERSION=6.6.0 TEST_DEBUG=true
-rvm:
-- jruby-1.7.25
-matrix:
-  include:
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=master
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=7.x
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.7
-  - rvm: jruby-1.7.27
-    env: LOGSTASH_BRANCH=5.6
-  - rvm: jruby-1.7.27
-    env: DISTRIBUTION=legacy INTEGRATION=true ES_VERSION=5.6.13 LOGSTASH_BRANCH=5.6
-      TEST_DEBUG=true
-  - rvm: jruby-9.1.13.0
-    env: DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-6 LOGSTASH_BRANCH=6.7
-      TEST_DEBUG=true
-  - rvm: jruby-9.1.13.0
-    env: DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-7 ARCH=LINUX
-      LOGSTASH_BRANCH=7.x TEST_DEBUG=true
-  - rvm: jruby-9.1.13.0
-    env: DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-8 ARCH=LINUX
-      LOGSTASH_BRANCH=master TEST_DEBUG=true
-  allow_failures:
-  - env: DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-8 ARCH=LINUX
-      TEST_DEBUG=true
-  - rvm: jruby-9.1.13.0
-    env: DISTRIBUTION=default INTEGRATION=true ES_VERSION=LATEST-SNAPSHOT-8 ARCH=LINUX
-      LOGSTASH_BRANCH=master TEST_DEBUG=true
-  fast_finish: true
-install: true
-script: ci/build.sh
-jdk: openjdk8
-before_install: gem install bundler -v '< 2'
+  - INTEGRATION=false ELASTIC_STACK_VERSION=5.x
+  - INTEGRATION=false ELASTIC_STACK_VERSION=6.x
+  - INTEGRATION=false ELASTIC_STACK_VERSION=7.x
+  - DISTRIBUTION=default INTEGRATION=true ELASTIC_STACK_VERSION=6.x
+  - DISTRIBUTION=default INTEGRATION=true ELASTIC_STACK_VERSION=7.x
+  - DISTRIBUTION=default INTEGRATION=true ELASTIC_STACK_VERSION=7.x SNAPSHOT=true
+  - DISTRIBUTION=default INTEGRATION=true ELASTIC_STACK_VERSION=8.x SNAPSHOT=true
```

Comment:
Just making sure that it is intentional to fail the builds to surface issues ASAP for visibility due to `8.x` incompatibilities given that we are so early in the `8.x` release cycle.

Reply:
It is intentional indeed, allow failures setting in travis can be useful but more often not if I see those fail I'll start caring less and less about them until we're on top of the release, may as well have these failures up front